query-engine/js-connectors/js/pg-js-connector/src/pg.ts

```diff
@@ -10,110 +10,115 @@ export type PrismaPgConfig = ConnectorConfig
 type StdClient = pg.Pool
 type TransactionClient = pg.PoolClient
 
-class PgQueryable<ClientT extends StdClient | TransactionClient> implements Queryable {
-    readonly flavour = 'postgres'
-    constructor(protected readonly client: ClientT) {
-    }
-
-    /**
-     * Execute a query given as SQL, interpolating the given parameters.
-     */
-    async queryRaw(query: Query): Promise<ResultSet> {
-        const tag = '[js::query_raw]'
-        debug(`${tag} %O`, query)
+class PgQueryable<ClientT extends StdClient | TransactionClient>
+  implements Queryable {
+  readonly flavour = 'postgres'
 
-        const { fields, rows: results } = await this.performIO(query)
+  constructor(protected readonly client: ClientT) {
+  }
 
-        const columns = fields.map(field => field.name)
-        const resultSet: ResultSet = {
-            columnNames: columns,
-            columnTypes: fields.map(field => fieldToColumnType(field.dataTypeID)),
-            rows: results.map(result => columns.map(column => result[column])),
-        }
+  /**
+   * Execute a query given as SQL, interpolating the given parameters.
+   */
+  async queryRaw(query: Query): Promise<ResultSet> {
+    const tag = '[js::query_raw]'
+    debug(`${tag} %O`, query)
 
-        return resultSet
-    }
+    const { fields, rows: results } = await this.performIO(query)
 
-    /**
-     * Execute a query given as SQL, interpolating the given parameters and
-     * returning the number of affected rows.
-     * Note: Queryable expects a u64, but napi.rs only supports u32.
-     */
-    async executeRaw(query: Query): Promise<number> {
-        const tag = '[js::execute_raw]'
-        debug(`${tag} %O`, query)
-
-        const { rowCount } = await this.performIO(query)
-        return rowCount
+    const columns = fields.map((field) => field.name)
+    const resultSet: ResultSet = {
+      columnNames: columns,
+      columnTypes: fields.map((field) => fieldToColumnType(field.dataTypeID)),
+      rows: results.map((result) => columns.map((column) => result[column])),
     }
 
-    /**
-     * Run a query against the database, returning the result set.
-     * Should the query fail due to a connection error, the connection is
-     * marked as unhealthy.
-     */
-    private async performIO(query: Query) {
-        const { sql, args: values } = query
-
-        return await this.client.query(sql, values)
-    }
+    return resultSet
+  }
+
+  /**
+   * Execute a query given as SQL, interpolating the given parameters and
```

Comment:
The parameters aren't interpolated in quaint/tokio_postgres — is it different in PG?

Reply:
We should fix that, at some point.